frontend/package.json

```diff
@@ -32,6 +32,7 @@
     "clsx": "1.2.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-equal-height": "^1.2.2",
```

Comment:
I like it! I would even recommend it to the colleague who's struggling w/ the same issue